crux-core/src/crux/backup.clj

```diff
@@ -12,13 +12,11 @@
 
 (s/def ::node-options
   (s/keys :req-un [:crux.kv/db-dir]
-          :gen    [:backup-dir]
-          :opt-un [:crux.standalone/event-log-dir]))
+          :gen    [:backup-dir]))
```

Comment:
spacing here is inconsistent with L19, prefer L19

Reply:
likewise - have been tidying up where I've seen it but missed this one, cheers